src/components/Navbar.js

```diff
@@ -0,0 +1,35 @@
+import Image from "next/image";
+import React from "react";
+
+const Navbar = ({ item1, item2 }) => {
+  return (
+    <>
+      <div>
+        <div className="flex flex-row justify-between items-center md:px-5">
+          <div className="relative w-32 aspect-video">
+            <Image
+              src="/AvantiFellowsLogo.svg"
+              alt="Avanti fellows logo"
+              fill
+            />
+          </div>
+          <p className={`md:text-2xl md:first-letter:text-3xl text-xs `}>
+            SESSION MANAGER
```

Comment:
These should be strings that are passed

Reply:
But items can be passed as strings where they are being imported

Reply:
I meant the string "SESSION MANAGER"